apps/opencs/view/doc/view.cpp

```diff
@@ -527,6 +550,59 @@ void CSVDoc::View::addSubView (const CSMWorld::UniversalId& id, const std::strin
 
     view->setStatusBar (mShowStatusBar->isChecked());
 
+    // Work out how to deal with additional subviews
+    //
+    // Policy for "Grow then Scroll":
+    //
+    // - Increase the horizontal width of the mainwindow until it becomes greater than or equal
+    //   to the screen (monitor) width.
+    // - Move the mainwindow position sideways if necessary to fit within the screen.
+    // - Any more additions increases the size of the mSubViewWindow (horizontal scrollbar
+    //   should become visible)
+    // - Move the scroll bar to the newly added subview
+    //
+    CSMSettings::UserSettings &settings = CSMSettings::UserSettings::instance();
+    QString mainwinScroll = settings.settingValue ("window/mainwindow-scrollbar");
+    if (mainwinScroll.isEmpty() || mainwinScroll == "Scrollbar Only")
+        mScrollbarOnly = true;
+    else if(mainwinScroll == "Grow Only")
```

Comment:
Yes.  That section was doing something earlier, but not needed any longer.  Will fix.